package.json

```diff
@@ -16,20 +16,20 @@
   "author": "Serverless, Inc.",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/aws-lambda": "^8.10.115",
-    "@types/chai": "^4.3.5",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^11.15.54",
-    "aws-lambda": "^0.1.2",
-    "chai": "^4.3.7",
-    "mocha": "^6.2.3",
-    "ts-mocha": "^6.0.0",
-    "ts-node": "^8.10.2",
-    "tslint": "^5.20.1",
-    "typescript": "^3.9.10"
+    "@types/aws-lambda": "8.10.24",
```

Comment:
Why dependencies are downgraded? 

Are those packages not respecting semver?

Reply:
`typescript` doesn't at the least. You were getting a build error because the versions are out of sync with each other. 

If we think we need to have the newer version then we should make a v2 that uses latest packages and node

Reply:
I simply followed semever and updated packages in version range that should not introduce breaking changes.

Majority of packages on npm follow semver, so it was pretty natural move, that normally works. It's kind of surprising it isn't here.

Therefore if you're sure that some of the bumped packages do not follow semver please fix them to a specific version

Reply:
@medikoo I have packages updated where it will build with the newest.